config/ca-trust/fwdproxy.crt

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is it ok to store this here or do we want it stored in param store>

Reply:
@LindseySaari It was in the k8s branch so I moved it over. 

@ericboehs thoughts?

Reply:
It [seems to be needed](https://github.com/department-of-veterans-affairs/va.gov-team/issues/41253) for the build process. 🤔